### PR TITLE
set-precedence

### PR DESCRIPTION
**Pull request by Amber1990Zhang**

### CODE REVIEW

**Comment by Amber1990Zhang** [comment]
> what is the precedence between pipe, set-ops, and Go ?

@dutor Please help.

**Comment by Amber1990Zhang** [comment]
> what is the precedence between pipe, set-ops, and Go ?


go > pipe > set

**Comment by whitewum** [comment]
> > what is the precedence between pipe, set-ops, and Go ?
> 
> go > pipe > set

This is what I actually want.

File: docs/manual-EN/2.query-language/2.functions-and-operators/set-operations.md [hunk 1]
```diff
@@ -168,3 +172,40 @@ returns
...
+
+For example:
+
+```ngql
+nebula> GO FROM 100 OVER follow YIELD follow._dst AS play_dst  \
+        UNION \
+        GO FROM 200 OVER serve REVERSELY YIELD serve._dst AS play_dst \
+        | GO FROM $-.play_dst OVER follow YIELD follow._dst AS play_dst;
+```
+
```
Comment (whitewum) [inline docs/manual-EN/2.query-language/2.functions-and-operators/set-operations.md:188]:
```
nebula> GO FROM 1 UNION GO FROM 2 | GO FROM 3;
```
It is the same as 
```
nebula> GO FROM 1 UNION (GO FROM 2 | GO FROM 3);
```

File: docs/manual-EN/2.query-language/2.functions-and-operators/set-operations.md [hunk 2]
```diff
@@ -168,3 +172,29 @@ returns
...
+![image](https://user-images.githubusercontent.com/42762957/84130415-d46c8a00-aa75-11ea-8a29-b8bef5e1d55f.png)
+
+The statement in the red bar is executed first, and then the statement in the green box is executed.
```
Comment (whitewum) [inline docs/manual-EN/2.query-language/2.functions-and-operators/set-operations.md:191]: are executed first concurrently.

**Pull request merged**